qiskit/pulse/instructions/play.py

```diff
@@ -30,26 +30,30 @@ class Play(Instruction):
     cycle time, dt, of the backend.
     """
 
-    def __init__(self, pulse: 'Pulse', channel: PulseChannel, name: Optional[str] = None):
+    def __init__(self, pulse,  # TODO: ``Pulse`` type annotation when cyclic import is removed
+                 channel: PulseChannel,
+                 name: Optional[str] = None):
         """Create a new pulse instruction.
 
         Args:
-            pulse: A pulse waveform description, such as
-                   :py:class:`~qiskit.pulse.pulse_lib.SamplePulse`.
+            pulse (qiskit.pulse.Pulse): A pulse waveform description, such as
+                                        :py:class:`~qiskit.pulse.pulse_lib.SamplePulse`.
             channel: The channel to which the pulse is applied.
             name: Name of the instruction for display purposes.
         """
         self._pulse = pulse
         self._channel = channel
-        super().__init__(pulse.duration, channel, name=name if name is not None else pulse.name)
+        if name is None:
+            name = pulse.name if pulse.name is not None else 'play'
+        super().__init__(pulse.duration, channel, name=name)
 
     @property
-    def operands(self) -> List[Union['Pulse', PulseChannel]]:
-        """Return a list of instruction operands."""
-        return [self.pulse, self.channel]
+    def operands(self) -> Tuple[Any, PulseChannel]:
+        """Return instruction operands: ``(Pulse, PulseChannel)``."""
+        return (self.pulse, self.channel)
 
     @property
-    def pulse(self) -> 'Pulse':
+    def pulse(self):  # -> Pulse
```

Comment:
Should this comment be here?

Reply:
I thought it was better than nothing, but maybe not. It does look like commented out code, which is bad. Can't do type hints here because of cylic imports. I should do `Return` documentation instead of this